pkg/WORKSPACE

```diff
@@ -23,6 +23,10 @@ load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
 
 bazel_skylib_workspace()
 
+### INTERNAL ONLY
+#
+# Include dependencies which are only needed for development here.
```

Comment:
It's not 100% clear that there isn't a `END INTERNAL ONLY` or something like that. Maybe add a `lines after this are not included in the release version of the WORKSPACE file`?

Reply:
Having an END would mean more chance for pilot error (and certainly more chance for sed error).
But +1 to improving the comment.  PTAL

Reply:
Sorry to not be clear. I think deleting to the end of file is definitely the right way to go.

Reply:
OK. Then we are on the same page.